lib/core/src/lib/viewer/components/viewer.component.ts

```diff
@@ -300,15 +301,20 @@ export class ViewerComponent<T> implements OnDestroy, OnInit, OnChanges {
     ) {}
 
     ngOnChanges(changes: SimpleChanges) {
-        const { blobFile, urlFile } = changes;
+        const { blobFile, urlFile, mimeType } = changes;
 
         if (blobFile?.currentValue) {
             this.mimeType = blobFile.currentValue.type;
+            this.mimeTypeIconUrl = this.thumbnailService.getMimeTypeIcon(mimeType.currentValue);
```

Comment:
I guess it should use either blobFile.currentValue.type or this.mimeType, instead of mimeType, since we want to cover the case where only the blobFile property was changed. But other than that, everything looks good to me.

Reply:
yes, updated the code, thanks

```
this.mimeTypeIconUrl = this.thumbnailService.getMimeTypeIcon(blobFile.currentValue.type);
```